.eslintrc.json

```diff
@@ -0,0 +1,39 @@
+{
+  "env": {
+    "browser": true,
+    "es2020": true,
```

Comment:
currently set to accept es2020 (ecmaVersion 11) syntax. This is probably fine, since the TypeScript compiler will compile down to the target ES version appropriate for browsers.